### PR TITLE
Tutorial: access review of assignments to privileged roles

### PR DESCRIPTION
**Pull request by FaithOmbongi**

### CODE REVIEW

File: concepts/tutorial-accessreviews-roleassignments.md [hunk 1]
```diff
@@ -0,0 +1,659 @@
...
+                "type": "user",
+                "userPrincipalName": "AdeleV@M365x010717.OnMicrosoft.com",
+                "lastUserSignInDateTime": ""
+            }
+        }
+    ]
+}
+```
+
+You can now record or submit your decisions for the access review instances.
```
Comment (isabelleatmsft) [inline concepts/tutorial-accessreviews-roleassignments.md:392]:
```suggestion
As the reviewer, you can now submit your decisions for the access review instance.
```

File: concepts/tutorial-accessreviews-roleassignments.md [hunk 2]
```diff
@@ -0,0 +1,659 @@
...
++ The reviewedBy object contains your details as the reviewer.
++ The applyResult is `New` meaning the decisions haven't been applied.
+
+Even though you've recorded all the decisions that were pending for this accessReviewInstance, these decisions haven't been applied to the resource and principal objects. For example, Adele still has access to the privileged role. This is because the **autoApplyDecisionsEnabled** was set to `false`, you haven't stopped the review, nor has the accessReviewInstance period expired and autocompleted.
+
+In this tutorial, you won't stop the accessReviewInstance manually but, you'll let the accessReviewInstance expire and autocomplete and then apply the access review decisions.
+
+## Step 5: Apply access review decisions
+
+As an admin, after the **status** of the accessReviewInstance is set to `Completed`, you can apply the access review decisions.
```
Comment (isabelleatmsft) [inline concepts/tutorial-accessreviews-roleassignments.md:457]:
```suggestion
As an admin, after the 3 day duration has passed and the **status** of the accessReviewInstance is set to `Completed`, you can apply the access review decisions.
```

File: concepts/tutorial-accessreviews-roleassignments.md [hunk 3]
```diff
@@ -0,0 +1,659 @@
...
+```http
+GET https://graph.microsoft.com/v1.0/identityGovernance/accessReviews/definitions/a13c348b-dc1c-47b5-8c58-b0d12b35a18b/instances/f282e54c-4eed-47a4-be9a-da00ab38aa8f/decisions
+```
+
+### Response
+
+The following response object is different from the response object you received in Step 3 with the following settings:
++ The access review decision for IT Helpdesk is Approve while for Adele is Deny
++ The reviewedBy object contains your details as the reviewer
++ The applyResult is New meaning the decisions haven't yet been applied. You must manually apply the decisions.
```
Comment (isabelleatmsft) [inline concepts/tutorial-accessreviews-roleassignments.md:488]: But didn't we just apply the decisions in the step above?

File: concepts/tutorial-accessreviews-roleassignments.md [hunk 4]
```diff
@@ -0,0 +1,659 @@
...
+            }
+        }
+    ]
+}
+```
+
+<!---
+## Step 6: Retrieve access review history definitions
+
+Contoso's auditors also want to review the access review history for the last quarter. In this example, you'll generate an access review history report for all acecssReviewScheduleDefinitions scoped to directory role assignments (roleAssignmentScheduleInstances). In this query, the **decisions** property is empty and therefore defaults to include all decisions in the history report.
```
Comment (isabelleatmsft) [inline concepts/tutorial-accessreviews-roleassignments.md:568]:
```suggestion
Contoso's auditors also want to review the access review history for the last quarter. In this example, you'll generate an access review history report for all accessReviewScheduleDefinitions scoped to directory role assignments (roleAssignmentScheduleInstances). In this query, the **decisions** property is empty and therefore defaults to include all decisions in the history report.
```

File: concepts/tutorial-accessreviews-roleassignments.md [hunk 5]
```diff
@@ -0,0 +1,659 @@
...
+                "type": "user",
+                "userPrincipalName": "AdeleV@M365x010717.OnMicrosoft.com",
+                "lastUserSignInDateTime": ""
+            }
+        }
+    ]
+}
+```
+
+<!---
```
Comment (isabelleatmsft) [inline concepts/tutorial-accessreviews-roleassignments.md:565]: i think we should definitely include this part

**Review by isabelleatmsft — CHANGES_REQUESTED** [review]
couple of minor changes needed, but overall looks great and will be super valuable to customers!

File: concepts/tutorial-accessreviews-roleassignments.md [hunk 6]
```diff
@@ -0,0 +1,659 @@
...
+## Step 1: Create an access review of privileged role assignments
+
+In this tutorial, we create a recurring access review of both *active* and *eligible* assignments to the User Administrator role. An **accessReviewScheduleDefinition** can be used to define the access review of multiple principle types (users and groups, or service principals) to only one privileged role. To review access to multiple privileged roles, create separate accessReviewScheduleDefinition objects.
+
+The following access review schedule definition has the following settings:
+
++ The principals (**principalScopes**) whose access will be reviewed are groups and users, and the resource (**resourceScopes**) is the User Administrator role.
++ The scope (**resourceScopes**) of the review is on both active and eligible User Administrator assignments.
++ You as the user in an Identity Governance Administrator role will be the reviewer.
++ The user in the Global Administrator role is the fallback reviewer. If the reviewers haven't recorded decisions when the review instance nears expiry, the fallback reviewer is notified of the pending access review for action.
```
[the remark this answers is not in the document]
Comment (FaithOmbongi) [reply concepts/tutorial-accessreviews-roleassignments.md:45]: Well noted. Applied the change. Though I'll ping you offline to understand this better.

File: concepts/tutorial-accessreviews-roleassignments.md [hunk 7]
```diff
@@ -0,0 +1,659 @@
...
++ The default decision is `Deny` when the main and fallback reviewers don't respond to the access review request before the instance expires.
++ **autoApplyDecisionsEnabled** isn't set and defaults to `false`. In this case, after the review completes, the decisions aren't automatically applied but you must manually apply them.
++ The review recurs every three months over a period of three days and doesn't end.
+
+### Request
+
+In the following request, replace the following values:
+
++ `4562bcc8-c436-4f95-b7c0-4f8ce89dca5e` with the value of your user identifier.
++ `77f8b7b6-1119-4a86-b4c3-e7d68e990424` with the value of the global administrator's user identifier.
```
[the remark this answers is not in the document]
Comment (FaithOmbongi) [reply concepts/tutorial-accessreviews-roleassignments.md:56]: On it locally.

File: concepts/tutorial-accessreviews-roleassignments.md (hunk 3 above)
Comment (FaithOmbongi) [reply concepts/tutorial-accessreviews-roleassignments.md:468]: Q: Good catch. Hadn't run the query for applying decisions for Adele.

File: concepts/tutorial-accessreviews-roleassignments.md (hunk 5 above)
Comment (FaithOmbongi) [reply concepts/tutorial-accessreviews-roleassignments.md:545]: I think it best that we use the recent changes thanks to Levi's PR? Let me retest.

File: concepts/tutorial-accessreviews-roleassignments.md [hunk 8]
```diff
@@ -0,0 +1,662 @@
...
+Contoso Limited is a growing service provider that has delegated various Azure AD administrator privileges to users, groups, and service principals in the organization. The company needs to ensure only the right assignees have access to privileged roles. The system auditors should also audit the access review history to report on the effectiveness of Contoso's internal controls.
+
+In this tutorial, you'll use the access reviews API to create a recurring access review of users and groups with access to privileged roles. This access includes both active and eligible roles.
+
+## Prerequisites
+
+To complete this tutorial, you need the following resources and privileges:
+
++ A working Azure AD tenant with an Azure AD Premium P2 or EMS E5 license enabled.
++ Sign in to [Graph Explorer](https://developer.microsoft.com/graph/graph-explorer) as a user in an Identity Governance Administrator role.
```
Comment (isabelleatmsft) [inline concepts/tutorial-accessreviews-roleassignments.md:20]:
```suggestion
+ Sign in to [Graph Explorer](https://developer.microsoft.com/graph/graph-explorer) as a user in a Privileged Role Administrator role.
```

File: concepts/tutorial-accessreviews-roleassignments.md [hunk 9]
```diff
@@ -0,0 +1,662 @@
...
++ Principals assigned to a privileged role that is the scope of your access review. To assign privileged roles, see [Tutorial: Use the Privileged Identity Management (PIM) API to assign Azure AD roles](/graph/tutorial-assign-azureadroles).
```
Comment (isabelleatmsft) [inline concepts/tutorial-accessreviews-roleassignments.md:21]:
```suggestion
+ Principals assigned to a privileged role. These assignments will be the scope of your access review. To assign privileged roles, see [Tutorial: Use the Privileged Identity Management (PIM) API to assign Azure AD roles](/graph/tutorial-assign-azureadroles).
```

File: concepts/tutorial-accessreviews-roleassignments.md [hunk 10]
```diff
@@ -0,0 +1,659 @@
...
+                "@odata.type": "#microsoft.graph.accessReviewQueryScope",
+                "query": "/users",
+                "queryType": "MicrosoftGraph"
+            },
+            {
+                "@odata.type": "#microsoft.graph.accessReviewQueryScope",
+                "query": "/groups",
+                "queryType": "MicrosoftGraph"
+            }
+        ],
```
[the remark this answers is not in the document]
Comment (isabelleatmsft) [reply concepts/tutorial-accessreviews-roleassignments.md:78]: bump @shubhamguptacal !

File: concepts/tutorial-accessreviews-roleassignments.md [hunk 11]
```diff
@@ -0,0 +1,662 @@
...
+
+```http
+GET https://graph.microsoft.com/v1.0/identityGovernance/accessReviews/definitions/a13c348b-dc1c-47b5-8c58-b0d12b35a18b/instances/f282e54c-4eed-47a4-be9a-da00ab38aa8f/decisions
+```
+
+### Response
+
+The following response object is different from the response object you received in Step 3 with the following settings:
++ The access review decision for IT Helpdesk is Approve while for Adele is Deny
++ The reviewedBy object contains your details as the reviewer
```
Comment (isabelleatmsft) [inline concepts/tutorial-accessreviews-roleassignments.md:464]: Also add that the deny result on Adele as applied, so the appliedBy property is filled on her decision

**Review by isabelleatmsft — APPROVED** [review]
Looks great! Approving with a couple small comments

**Pull request merged**